### DISCUSSION TITLE
Support for more printer in orcaslicer

### DISCUSSION THREAD

**Discussion by hanzi2u** [OrcaSlicer/OrcaSlicer#634] [Ideas]
hello i am writing to hear if it isn't able to add more printer to the list when installing more printer in the software , here specifically i'm talking about the elegoo brand, i do myself have an elegoo neptune 3 plus, and i just thought it could be neat if there was a way to add that to the slicer , i'm running klipper on the machine

**Comment by TexZeTech** [comment]
So you can create a profile and work off that but as far as I know at this point in time, the profile doesn't exist so you need to make a base profile from scratch and then submit it to BambuSlicer/OrcaSlicer respective git pages. 

I based the printer I'm using off the Voron trident profile then just removed all that wasn't relevant and renamed everything. 

Because I'm using the appimage I can only add the custom profile in the config section and not the base install directory (curses in appimage) 

that's my 2 cents. Good luck with your quest for the printer profile.

**Reply by TexZeTech** [reply]
I think there is also a builder but I haven't used that.
